hack/boilerplate/boilerplate.go

```diff
@@ -28,9 +28,9 @@ import (
 )
 
 var (
-	boilerplatedir = flag.String("boilerplate-dir", ".", "Boilerplate directory for boilerplate files")
-	rootdir        = flag.String("rootdir", "../../", "Root directory to examine")
-	verbose        = flag.Bool("v", false, "Verbose")
+	boilerplateDir = flag.String("boilerplate-dir", ".", "Directory containing boilerplate files")
+	rootDir        = flag.String("root-dir", "../../", "Root directory to examine")
```

Comment:
You can't change flags, without also changing the invocation. Preferably, the old name should be kept (and hidden) for backwards compatibility. This includes keeping the short option, when adding a new long option.